src/index.html

```diff
@@ -11,12 +11,60 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
+    <header class="header">
+      <a
+        class="logo__link"
+        href="index.html"
+      >
+        <img
+          src="./images/moyo_1.svg"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list__item">
+            <a class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__list__item">
+            <a class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list__item">
+            <a class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__list__item">
+            <a
+              class="nav__link"
+              data-qa="hover"
```

Comment:
The `data-qa` attribute is used here, which is typically for testing purposes. Ensure that this attribute is necessary and used consistently across your project if it's part of your testing strategy. If not, consider removing it to keep the markup clean.